examples/aws/10-deployment-kube2iam.yaml

```diff
@@ -42,6 +42,8 @@ spec:
             - /ark
           args:
             - server
+            - --backup-sync-period
```

Comment:
I think we can probably skip explicitly specifying this flag in all of the example YAMLs.  Not sure why we had the value explicitly set to something other than the default value - likely not intentional. @ncdc any reason you can recall? (the default value for backup sync is `60m` in code).

Reply:
I propose we leave it for the purpose of an example how to correct pass in flags for the server. I think this has value even tho changing this value doesn't seem useful.